test/FunctionalTests/Microsoft.OData.Core.Tests/ScenarioTests/UriParser/FilterAndOrderByFunctionalTests.cs

```diff
@@ -2830,6 +2830,42 @@ public void FilterWithInOperationWithOpenTypes()
             Assert.Equal("('examplepainting')",
                 Assert.IsType<CollectionConstantNode>(inNode.Right).LiteralText);
         }
+
+        [Theory]
+        [InlineData("example in ('')", "\"\"")] // No space
+        [InlineData("example in (' ')", " ")] // 1 space
+        [InlineData("example in ( '   ' )", "   ")] // 3 spaces
+        [InlineData("example in ( \"  \" )", "  ")] // 2 spaces
+        [InlineData("example in ( \"    \" )", "    ")] // 4 spaces
+        public void FilterWithInOperationWithOpenTypesInEmptyString(string filterQueryString, string expectedLiteral)
+        {
+            FilterClause filter = ParseFilter(filterQueryString,
+                HardCodedTestModel.TestModel, HardCodedTestModel.GetPaintingType());
+
+            var inNode = Assert.IsType<InNode>(filter.Expression);
+            CollectionConstantNode collectionNode = Assert.IsType<CollectionConstantNode>(inNode.Right);
+            Assert.Equal(1, collectionNode.Collection.Count);
+
+            ConstantNode constantNode = collectionNode.Collection.First();
+            Assert.Equal(expectedLiteral, constantNode.LiteralText);
+        }
+
+        [Theory]
+        [InlineData("example in ('', \"  \")")]
+        [InlineData("example in (' ', \"\")")]
+        [InlineData("example in ( '   ', '' )")]
+        [InlineData("example in ( \"  \", \"\" )")]
+        [InlineData("example in ( \"    \", ' ' )")]
+        public void FilterWithInOperationWithOpenTypesInMultipleEmptyStrings(string filterQueryString)

```

Comment:
what are the expected types on these multiple empty strings?

Reply:
Added